server/services/store/sqlstore/migrate.go

```diff
@@ -176,9 +176,9 @@ func (s *SQLStore) Migrate() error {
 	}
 
 	var mutex *cluster.Mutex
-	if s.pluginAPI != nil {
+	if s.IsPlugin {
 		var mutexErr error
-		mutex, mutexErr = cluster.NewMutex(s.pluginAPI, "Boards_dbMutex")
+		mutex, mutexErr = s.NewMutexFn("Boards_dbMutex")
```

Comment:
This is always defined in this context, but you don't need to know that. So, if you receive 2 parameters, they should be checked independently, in this case something like, `if s.IsPlugin and s.NewMutexFn != nil` and you have to modify the other calls using the mutex too.